packages/notebook-extension/src/index.ts

```diff
@@ -1702,11 +1704,9 @@ function addCommands(
         return;
       }
 
-      return Private.createConsole(
-        commands,
-        current,
-        args['activate'] as boolean
-      );
+      return Private.createConsole(commands, current, args[
+        'activate'
+      ] as boolean);
```

Comment:
@mlucool I think these are the lines that offending the lint check. Since you didn't change anything here other than formatting, putting these lines back the way they were should fix the lint check

Reply:
Fixed...not sure what happened with lint. Thanks!